tutorials/mongoose/query-string.md

```diff
@@ -0,0 +1,25 @@
+Mongoose's `Model.find()` function is an important method to understand.
+You may call it without any arguments and it will return all the documents in that model.
+You may hone the search by passing in a `filter` which tells mongoose what to look for in the database.
+This `filter` may be an `objectId` or an `object`.
+When using `Model.find()`, you should explicitly list the parameters you are searching for in the model.
```

Comment:
This is important when pulling filter parameters from a query string, but not necessarily important in all cases.